### ISSUE TITLE
Invert checkboxes

### ISSUE THREAD

**Issue by Sav22999** [Mte90/Share-Backported#147]
Instead of select checkboxes you want to disable it would be better select those you want to enable

**Comment by Mte90** [comment]
It is now in this way because when we add a new service with a release the new service is automatically shown.
In this way the users can see if there are new services without check an update pages and disable it.
If we move further this we need to inform the user of that.

**Comment by Sav22999** [comment]
Yes, but it's not logically correct. When you insert a new service, just insert is as "checked" instead of "unchecked"

**Comment by Mte90** [comment]
How you can detect that is a new service so with the update become checked automatically?
We don't have any way to execute a code on upgrade, and we save just the value as they are now in the db.

**Comment by Ryuno-Ki** [comment]
> How you can detect that is a new service so with the update become checked automatically?

By checking its presence in the storage. If it's not there, it wasn't present on the last save. Thus, it is likely a new service.
This change could be considered a breaking change. Perhaps worth for reserving after the next major release?

**Comment by Sav22999** [comment]
@Mte90 Or just you set "checked" as default. It's the same of the unchecked logic

**Comment by Ryuno-Ki** [comment]
It's breaking insofar, as the logic (the meaning of checked) is flipped. Thus, it has to be communicated somehow to users upgrading to the new version.

**Comment by Mte90** [comment]
> It's breaking insofar, as the logic (the meaning of checked) is flipped. Thus, it has to be communicated somehow to users upgrading to the new version.

This is the big problem as the extension is built to be simple so to do that we need to implement a lot of stuff.

To implement this we need to implement a backport solution that doesn't break anything.

**Comment by Mte90** [comment]
I don't know if it is the case to close that ticket as after years I didn't got any support request about it.

**Comment by Sav22999** [comment]
Do you pay to have opened issues? 🤔
Just leave open, imho, and if a user want yo contribute, then he/she/them will be it

**Comment by Mte90** [comment]
It was just a way to updated the ticket to get people find it for hacktoberfest 😂
